index.html

```diff
@@ -3,43 +3,314 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>National Homlessness</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
       text-align: center;
-    }
+            } 
+    h1      {
+      color:blue;
+      font-family: "Trebuchet MS";
+      font-size: 300%;
+            } 
+    table {border: 1px solid black;}
+    tr:nth-child(even) {background-color:#f5f5f5}
+    th      { 
+      background-color: green;
+      color: white;
+            }
   </style>
 </head>
 <body>
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <h1>Change in Overall Homelessness, 2013-2014</h1>
+      <a href="http://www.endhomelessness.org/page/-/files/State_of_Homelessness_2015_FINAL_online.pdf">Source: National Alliance to End Homelessness</a>
```

Comment:
Instead of leaving this A tag hanging out on the page, it would be better to wrap it in a P (paragraph) tag. One additional benefit to that would be that it'll then automatically have a `margin-bottom` to increase the space between it and the table below.